mautic/mautic
# Add support to dispatch CampaignEvents::ON_EVENT_SCHEDULED_BATCH for the batch of failed rescheduled jobs

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | enhancement
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

This simply dispatches the CampaignEvents::ON_EVENT_SCHEDULED_BATCH event for the batch of failed logs for listeners that want to process the reschedules in batch rather than one by one. 

It includes #6207

[//]: # ( As applicable: )
#### Steps to test this PR:
1. Mainly make sure that failed events get rescheduled
2. Create a campaign to send an email
3. Unpublish the email
4. Execute the campaign
5. The send email job should be rescheduled for an hour later for each contact (or whatever you have set as your failed campaign event interval in Configuration -> Campaign Settings
